src/services/stats/cras/durees.js

```diff
@@ -32,7 +32,7 @@ const getStatsDurees = async (db, query) => {
               $gte: 60,
               $lt: 120,
             } },
-            { 'cra.duree': { $eq: '90' } } //Correspond au bouton 1h30 sans précision de duree
+            { 'cra.duree': { $eq: '60' } } //Correspond au bouton 1h pile
```

Comment:
Tu avais vu juste => $in: ['60, '90'] comme ca pas de soucis pour les CRA V1 qui avaient une valeur par défaut à "60" et V2 avec par défaut "90"